test/auth.js

```diff
@@ -39,19 +39,25 @@ describe('auth interface', () => {
       .then((resp) => should(resp.uri).startWith('safe-auth:'));
   });
 
-  // TODO: issue MAID-2542 has been raised for safe_app lib as it's panicking
-  // instead of returning an error code and message.
-  // We'll need to adapt the test accordingly nce that's fixed.
   it('throws error if permissions object contains invalid permission', async () => {
     const app = await h.createTestApp();
     const test = () => app.auth.genAuthUri({ _public: ['Invalid'] });
     return should(test).throw('Invalid is not a valid permission');
   });
 
-  it('should throw error if non-standard container is requested', () => {
+  it('should throw error if invalid container permissions format is passed', () => {
     const containersPermissions = { _app: ['Read', 'Insert', 'ManagePermissions'] };
     return should(createAuthenticatedTestApp('_test_scope', containersPermissions, { own_container: true }))
-      .be.rejected();
+      .be.rejectedWith("'_app' not found in the access container");
+  });
+
+  it('should throw error if non-standard container is requested', () => {
```

Comment:
I don't think this test adds anything to the previous one, we could keep this one and remove the previous perhaps, but both seem redundant to me.